rasa/jupyter.py

```diff
@@ -13,14 +13,25 @@ def pprint(object: Any):
 
 
 def chat(model_path: Text = None, agent: 'Agent' = None,
-         interpreter: NaturalLanguageInterpreter = None):
+         interpreter: NaturalLanguageInterpreter = None) -> None:
+    """Chat to the bot within a Jupyter notebook.
+
+    Args:
+        model_path: Path to a Rasa Stack model.
+        agent: Rasa Core agent (used if no Rasa Stack model given).
+        interpreter: Rasa NLU interpreter (used with Rasa Core agent if no
+                     Rasa Stack model is given).
+    """
 
     if model_path:
         from rasa.run import create_agent
         unpacked = model.get_model(model_path)
         agent = create_agent(unpacked)
     elif agent and interpreter:
-        agent.interpreter = NaturalLanguageInterpreter.create(interpreter)
```

Comment:
let's add a comment here that we are aware this is not a proper path and a hack.